sklearn/ensemble/tests/test_voting.py

```diff
@@ -124,6 +163,15 @@ def test_weights_iris():
     scores = cross_val_score(eclf, X, y, cv=5, scoring='accuracy')
     assert_almost_equal(scores.mean(), 0.93, decimal=2)
 
+    reg1 = BaggingRegressor(base_estimator=DecisionTreeRegressor(),
+                            random_state=1, n_estimators=100)
+    reg2 = RandomForestRegressor(random_state=1, n_estimators=100)
+    reg3 = GradientBoostingRegressor(random_state=1, n_estimators=100)
+    ereg = AveragingRegressor([('br', reg1), ('rf', reg2), ('gbr', reg3)],
+                              weights=[1, 2, 10])
+    scores = cross_val_score(ereg, X_r, y_r, cv=5, scoring='r2')
+    assert_almost_equal(scores.mean(), 0.67, decimal=2)
```

Comment:
It's quite unusual to put tests for different estimators in the same function, unless:
* You are testing for identical behaviours between them
* You parameterize the test to account for small differences in their behaviour

You could do parameterize here if you like.